01_01_Basic_Rules.txt

```diff
@@ -513,9 +513,27 @@ succeed at a check. The DC does not need to be given out by the GM, but if
 a DC seems too high, you have the right to grumble (but seriously, ask your GM if 
 something seems too difficult. There is usually a reason why tying your shoe 
 takes a DC of 70% on that particular day, etc). After you have rolled, please
-apply any and all relevant skills and Stat Bonuses to a roll, as instructed by your GM.
+apply any and all relevant skills and Stat Modifiers to a roll, as instructed by your GM.
 When rolling Investigation for example, you will likely add your PER Stat Bonus.
 
+	Critical successes occur when your roll on a skill check lands inside your
+Critical range (aka your critical chance). For skill checks, the base critical range
+is 10, meaning that a roll greater than 90 is a Critical success. Luck modifies your
+Critical range: your Luck modifier is added to your Critical range. For example, if
+you have a Luck modifier of 12, your Critical range would be 22, and rolls greater
+than 78 would result in a Critical success. Rolls that land in the upper half of your
+Critical range result in an Extra-Critical success; with a Luck modifier of 12, rolls
+greater than 89 would result in an Extra-Critical success. If your Luck modifier is
+less than -8, you cannot roll Critical successes, but rolls of exactly 100 result in
+Extra-Critical successes.
+
+	In general, if you roll a critical success on a skill check, add 10 to your
+total roll. For an extra critical, add 20.

```

Comment:
I think it would be appropriate to add a 'some abilities may trigger additional effects when you score criticals' like in the combat crit section.